wp-content/themes/reconasia/assets/_scss/print.scss

```diff
@@ -0,0 +1 @@
+@use 'modules/print';
```

Comment:
I don't think you need to use `@use` here - just copy the contents of `/modules/_print.scss` directly into `/_scss/print.scss`.